Cheat Engine/bin/celua.txt

```diff
@@ -1826,11 +1828,23 @@ properties
   IsReadable: boolean - Set to false if record contains an unreadable address. NOTE: This property will not be set until the value property is accessed at least once. (ReadOnly)
 
   Options: String set - a string enclosed by square brackets filled with the options seperated by a comma. Valid options are: moHideChildren, moActivateChildrenAsWell, moDeactivateChildrenAsWell, moRecursiveSetValue, moAllowManualCollapseAndExpand, moManualExpandCollapse
+  
+  DropDownLinked: boolean - if dropdown list refers to list of another memory record eg. (memrec name)
+  DropDownLinkedMemrec: string - Description of linked memrec or emptystring if not linked
+  DropDownList : StringList - self explanatory (returns new list each time)
+  DropDownReadOnly: boolean - true if 'Disallow manual user input' is set
+  DropDownDescriptionOnly: boolean - self explanatory
+  DisplayAsDropDownListItem: boolean - self explanatory
+  DropDownCount: integer - equivalent to .DropDownList.Count
+  DropDownValue[index] : Array to access values in DropDownList (read-only)
+  DropDownValue[index] : Array to access Descriptions in DropDownList (read-only)

```

Comment:
DropDownDescription : Array to access Descriptions in DropDownList (read-only)

Reply:
oops, corrected. Though I don't suppose it's possible to squash into a single commit via the website... didn't actually make a local repo just to edit celua.txt